angel_system/data/medical/load_bbn_data.py

```diff
@@ -230,38 +230,54 @@ def save_as_kwcoco(classes, data, save_fn="bbn-data.mscoco.json"):
 
 
 def activity_label_fixes(task, activity_label, target):
-    # print(activity_label, target)
+    print(f"task: {task}, target: {target}, activity_label: {activity_label}")
+    label = None
+    label_id = None
     if task == "m2":
-        if activity_label == "put_tourniquet_around":
+        if activity_label == "put_tourniquet_around" or activity_label == "open_up":
             label = "place-tourniquet"
             label_id = 1
-        if activity_label == "pulls_tight":
+        elif activity_label == "pulls_tight":
             label = "pull-tight"
             label_id = 2
-        if activity_label == "secures" and target == "velcro_strap":
+        elif activity_label == "secures" and target == "velcro_strap":
             label = "apply-strap-to-strap-body"
             label_id = 3
-        if activity_label == "twist" and target == "windlass":
+        elif activity_label == "twist" and target == "windlass":
             label = "turn-windless"
             label_id = 4
-        if (
+        elif (
             activity_label == "locks_into_windlass_keeper"
             or activity_label == "lock_into_windlass_keeper"
         ):
             label = "lock-windless"
             label_id = 5
-        if (
+        elif (
             activity_label == "wraps_remaining_strap_around"
             or activity_label == "wrap_remaining_strap_around"
         ):
             label = "pull-remaining-strap"
             label_id = 6
-        if activity_label == "secures" and target == "windlass":
+        elif activity_label == "secures" and target == "windlass":
             label = "secure-strap"
             label_id = 7
-        if activity_label == "writes_on" and target == "tourniquet_label":
+        elif activity_label == "writes_on" and target == "tourniquet_label":
             label = "mark-time"
             label_id = 8
+
+    elif task == "m3":
+        pass
+
+    elif task == "m4":
+        pass
```

Comment:
Nitpick: These don't *really* need to be here since there is no `else` statement that you're trying to skip. A comment would suffice to describe that there are no fixes for activity m3/m4 tasks.

Reply:
Did you mean to remove the `task == "m5"` condition that was here in your edit previously?

Reply:
Yes - we don't use this function for any of the tasks anymore. no need to map actions to steps if we already have the step ground truth (not sure why it was written to begin with).